tests/ccmcluster.py

```diff
@@ -105,12 +105,9 @@ def parse_cluster_status(self, stdout):
             return:
             [(node1, UP)]
         """
-        nodes_status = []
-        for line in stdout.split("\n")[2:]:
-            node, status = line.split(":")
-            nodes_status.append((node.strip(), status.strip()))
-
-        return nodes_status
+        output = stdout.split("\n")
```

Comment:
why are you touching the logic here ? for what end ?

Reply:
the stdout contains log messages with the froamt
```
2021-12-28 16:29:37,617 ccmlib.cluster DEBUG cluster.py:406 |Cluster: 'CCMCluster-cassandra'
2021-12-28 16:29:37,617 ccmlib.cluster DEBUG cluster.py:407 |-------------------------------
2021-12-28 16:29:37,618 ccmlib.node DEBUG node.py:294 |node1: UP
2021-12-28 16:29:37,618 ccmlib.node DEBUG node.py:294 |node2: UP)
DEBUG: 2021-12-28 16:29:37 (ccmcluster.py:90 [stderr] )
INFO: 2021-12-28 16:29:37 (ccmcluster.py:78 ['./ccm', 'node1', 'nodetool', 'status'])
DEBUG: 2021-12-28 16:29:38 (ccmcluster.py:89 [stdout] 2021-12-28 16:29:37,896 ccmlib.cluster DEBUG cluster.py:698 |Started cluster 'CCMCluster-cassandra' version 3.11.4 installed in /home/oren/.ccm/repository/3.11.4
2021-12-28 16:29:38,639 ccmlib.cmds.node_cmds DEBUG node_cmds.py:264 |
2021-12-28 16:29:38,639 ccmlib.cmds.node_cmds DEBUG node_cmds.py:265 |Datacenter: datacenter1
=======================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address    Load       Tokens       Owns (effective)  Host ID                               Rack
UN  127.0.0.1  98.97 KiB  1            100.0%            1620c074-52ee-48cb-bda1-19a975907f61  rack1
UN  127.0.0.2  113.2 KiB  1            100.0%            9b43f1f8-e4de-4c05-a47a-7efc22c0401c  rack1)
```

Reply:
please revert those change from the test, ccm working from commandline should be affect by those changes

Reply:
Done